packages/ra-ui-materialui/src/field/ReferenceArrayField.tsx

```diff
@@ -80,7 +103,22 @@ const useStyles = makeStyles(
     { name: 'RaReferenceArrayField' }
 );
 
-export const ReferenceArrayFieldView = ({
+type ReferenceArrayFieldClassKey = 'progress';
+
+interface ReferenceArrayFieldViewProps {
+    children: ReactElement;
+    classes?: Partial<ClassNameMap<ReferenceArrayFieldClassKey>>;
+    className: string;
+    data: RecordMap;
```

Comment:
not required (while loading, the data is empty)

Reply:
`useRefereneArrayFieldController` returns `data: RecordMap;` in its `ReferenceArrayProps`